include/UIKit/UIView.h

```diff
@@ -379,11 +379,10 @@ UIKIT_EXPORT_CLASS
 @end
 
 // WinObjC Xaml Extensions
-@class WXFrameworkElement;
 @interface UIView (UIKitXamlExtensions)
-+ (WXFrameworkElement*)createXamlElement;
-@property (nonatomic, readonly, strong) WXFrameworkElement* xamlElement;
-- (instancetype)initWithFrame:(CGRect)frame xamlElement:(WXFrameworkElement*)xamlElement;
++ (RTObject*)createXamlElement;
```

Comment:
Code inside the frameworks should not call xamlElement to get the underlying XAML element. Instead, it should call _xamlElementInternal, declared in UIViewInternal.h.

The xamlElement property is intended for use only by application code. It will work inside the frameworks, but it's less efficient and harder to use.